sso/sso.js

```diff
@@ -0,0 +1,109 @@
+const router = require('express').Router()
```

Comment:
Een heel aantal console.log'jes in de happy path in deze file

Reply:
Ja, maar dat helpt ontwikkelaars wel heel erg als ze hier hun eigen applicatie tegen aan ontwikkelen.

Reply:
True. Maar kan er dus binnenkort wel uit denk ik?